projects/arlas-toolkit/assets/i18n/en.json

```diff
@@ -125,90 +99,138 @@
-	"Missing permissions to create a dashboard": "Missing permissions to create a dashboard",
-	"A configuration with this name exists already, please choose another name": "A dashboard with this name exists already, please choose another name",
-	"An error occurred, please try later": "An error occurred, please try later",
-	"Create the permission": "Create the permission",
-	"Creating": "Creating ...",
-	"Loading data of": "Loading data of '{{collection}}'",
-	"Search...": "Search...",
+	"Sign in": "Sign in",
+	"Please enter your password": "Please enter your password",
+	"Forgot password?": "Forgot password?",
+	"Login or password invalid": "Login or password invalid",
+	"Login": "Log in",
+	"No Account?": "Don't have an Account?",
+	"x-axis": "x-axis",
+	"to": "to",
+	"y-axis": "y-axis",
+	"interval": "",
+	"If you have an account, you will receive an email shortly": "If you have an account, you will receive an email shortly",
+	"Send password reset email": "Send password reset email",
+	"Zoom to data": "Zoom to data",
+	"See more": "See more",
+	"Clear all filters": "Clear all filters",
+	"Actions": "Actions",
+	"View": "View",
+	"Remove": "Remove",
+	"Type": "Type",
+	"CSV": "CSV",
+	"Json": "Json",
+	"Important": "Important",
+	"This tool allows you to get a docker command line to download your data": "This tool allows you to get a docker command line to download your data",
+	"Make sure you have": "Make sure you have",
+	"installed before continuing": "installed before continuing",
+	"Collection": "Collection",
+	"Command line": "Command line",
+	"Target OS": "Target OS",
+	"Copy": "Copy",
+	"Copied": "Copied",
+	"Close": "Close",
+	"group": "Groups",
+	"reading": "Right to read",
+	"writing": "Right to write",
+	"Share": "Share",
+	"Cancel": "Cancel",
+	"Edit": "Edit",
+	"Duplicate": "Duplicate",
+	"Rename": "Rename",
+	"Delete": "Delete",
+	"This will delete": "This will delete",
+	"element and cannot be undone.": "element and cannot be undone",
+	"New name": "New name",
+	"Add a new dashboard": "Create new dashboard",
+	"Create": "Create",
+	"Share dashboard": "Share dashboard",
+	"Change password": "Change password",
+	"Old password": "Old password",
+	"Old password does not match": "Old password does not match",
+	"Add a dataset": "Add a bookmark",
+	"Manage datasets": "Manage bookmarks",
+	"Count": "Number of elements",
+	"Bookmarks can be combined to create a new one containing the filters of all selected bookmarks.": "Bookmarks can be combined to create a new one containing the filters of all selected bookmarks.",
+	"The color next to the name matches the combination of fields and operations of the filters.": "The color next to the name matches the combination of fields and operations of the filters.",
+	"Filters on the same field are combined with an OR, whereas for different fields it is with an AND.": "Filters on the same field are combined with an OR, whereas for different fields it is with an AND.",
```

Comment:
I'd say, bookmarks sharing the exact same fields and operators are combined with an OR, otherwise they are combined with an AND